system/jlib/jlog.hpp

```diff
@@ -1466,10 +1475,20 @@ struct LogAccessConditions
         timeRange = l.timeRange;
         setFilter(LINK(l.filter));
```

Comment:
style: It is not clear that setFilter takes an owned pointer.  A more recent convention is for setFilter to link the argument, or if there is a good reason to pass an owned pointer name the function setFilterOwn (e.g. see LogMsgMonitor).  I know this is rule is broken a lot....but mainly in constructors rather than set() functions

Reply:
I see your point, however this is previously established code and the change is (as you point out) more stylistic and perhaps improves maintainability. does that fit the criteria for a separate jira?